data/types/base.go

```diff
@@ -248,14 +249,14 @@ func (b *Base) IdentityFields() ([]string, error) {
 	if b.Time == nil {
 		return nil, errors.New("time is missing")
 	}
-	if *b.Time == "" {
+	if (*b.Time).IsZero() {
```

Comment:
I'm not entirely sure checking for `IsZero()` is necessary (or even semantically correct). The empty value of `b.Time` is `nil`, not a zero time.

Reply:
It's ok to leave as it is, because the original code had the check for an empty string